content/livesync/connector/database.textile

```diff
@@ -106,26 +106,30 @@ GRANT DELETE ON your_nodes_schema.your_nodes_table to your_user;
 
 h2(#outbox). Outbox table
 
-The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to send realtime, application-defined change events via Ably's global edge messaging platform. The outbox pattern enables you to reliably publish an ordered sequence of change event messages over Ably, providing exactly-once delivery guarantees.
+The Database Connector utilizes the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to send realtime, application-defined change events via Ably's global edge messaging platform. The outbox pattern allows for the reliable publication of an ordered sequence of change event messages over Ably, providing exact-once delivery guarantees. This pattern involves the use of an outbox table that you create in your database.
+The Database Connector automatically monitors new records written to the outbox table and publishes them as messages over "channels":/channels.
+When your backend server processes a request to update data in your database, it should also write a corresponding change event to the outbox table within the same transaction as the underlying mutation. The Database Connector then publishes the change event, written to the outbox, to the channel in the exact order that the underlying change to the data is committed to the database. Furthermore, the Database Connector automatically deletes records from the table once the records are successfully processed.
 
-The outbox pattern uses an outbox table that you create in your database.
-
-The Database Connector automatically observes new records written to the outbox table and publishes them as messages over "channels":/channels.
-
-When your backend server processes a request to update data in your database, it should write a corresponding change event to the outbox table in the same transaction as the underlying mutation:
+This example adds a comment and logs the event in an outbox:
 
 ```[text]
 BEGIN;
 -- mutate your data, e.g.:
 INSERT INTO comments (comment) VALUES ('New comment!');
 -- write change event to outbox, e.g.:
-INSERT INTO outbox (mutation_id, channel, name, ...) VALUES ('my-mutation-id', 'posts:123', 'addComment', ...);
+INSERT INTO outbox (mutation_id, channel, name, data) VALUES ('my-mutation-id', 'posts:123', 'addComment', 'New comment!');
 COMMIT;
 ```
 
-The Database Connector will publish the change event, written to the outbox, to the channel in the exact order in which the underlying change to the data is committed to the database.
+The following describes the ‘adds a comment and logs the event in an outbox’ example:
```

Comment:
Per previous comment, let's just describe the columns of the outbox table below. These should be documented elsewhere anyway, I think it would be better to just link to https://ably.com/docs/livesync/connector/database#schema

Reply:
See 3146122462d451ef6e0069c3a09b11dd3b4a566d